pkgs/bitte-shell.nix

```diff
@@ -0,0 +1,72 @@
+{ bitte
+, lib
+, writeText
+, mkShell
+, nixos-rebuild
+, terraform-with-plugins
+, scaler-guard
+, sops
+, vault
+, openssl
+, cfssl
+, nixfmt
+, awscli
+, nomad
+, consul
+, consul-template
+, python38Packages
+, direnv
+, nixFlakes
+, jq }:
+
+{ cluster
+, caCert ? null
+, domain
+, extraEnv ? {}
+, extraPackages ? []
+, region
+, profile
+, nixConfig ? null
+}: let
+
+in mkShell ({
+  # for bitte-cli
+  LOG_LEVEL = "debug";
+
+  NIX_CONFIG = ''
+        extra-experimental-features = nix-command flakes ca-references recursive-nix
+        extra-substituters = https://hydra.iohk.io
+        extra-trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+        bash-prompt = [bitte]
+  '' + (lib.optionalString (nixConfig != null) nixConfig);
+
+  BITTE_CLUSTER = cluster;
+  AWS_PROFILE = profile;
+  AWS_DEFAULT_REGION = region;
+  VAULT_ADDR = "https://vault.${domain}";
+  NOMAD_ADDR = "https://nomad.${domain}";
+  CONSUL_HTTP_ADDR = "https://consul.${domain}";
+
+  buildInputs = [
+    bitte
+    nixos-rebuild
+    terraform-with-plugins
+    scaler-guard
+    sops
+    vault
+    openssl
+    cfssl
+    nixfmt
+    awscli
+    nomad
+    consul
+    consul-template
+    python38Packages.pyhcl
```

Comment:
we don't use this.

Reply:
This was mostly copypasted from the old darwin branch so there must be some outdated stuff.
So remove `nomad`, `consul`, `consul-template` and whatever that python thing is?

Reply:
We can keep the others, just pyhcl won't be used.

Reply:
I was gonna say that I use the nomad cli quite a bit :laughing: